scripts/build/build/targets.py

```diff
@@ -127,6 +127,7 @@ def BuildHostTarget():
         TargetPart('simulated-app2', app=HostApp.SIMULATED_APP2,
                    enable_ble=False, enable_wifipaf=False),
         TargetPart('python-bindings', app=HostApp.PYTHON_BINDINGS),
+        TargetPart('python-bindings-webrtc', app=HostApp.PYTHON_BINDINGS_WEBRTC),
```

Comment:
You need to add ` target.AppendModifier('webrtc', enable_webrtc=True)` not `TargetPart`